discovery-infra/test_infra/consts/consts.py

```diff
@@ -1,6 +1,14 @@
+from enum import Enum
+
 from .durations import MINUTE, HOUR
 
 
+class OpenshiftVersion(Enum):
```

Comment:
what enum gives here? 
Why it can't be string that will not require adding .value?

Reply:
Those values can't be changed on runtime 
And I think @YuviGold suggested it